modules/qna/src/backend/storage.ts

```diff
@@ -164,13 +167,19 @@ export default class Storage {
     if (!question.data.answers) {
       question.data.answers = [question.data.answer]
     }
+    if (!question.data.lastModified) {
```

Comment:
this whole function should be removed.  as it inline (to comply with 11.3 ... which is totally outdated)


You should create a migration for this (see our wiki on creating migrations)